docs/tutorials/rerank/rerank_pipeline_with_Cohere_Rerank_model_on_Amazon_Bedrock.md

```diff
@@ -0,0 +1,665 @@
+        "_id": "3",
+        "_score": 0.7190094,
+        "fields": {
+          "passage_text": [
+            "Washington, D.C. (also known as simply Washington or D.C., and officially as the District of Columbia) is the capital of the United States. It is a federal district."
+          ]
+        },
+        "highlight": {
+          "passage_text": [
+            "(also known as simply Washington or D.C., and officially as <strong>the</strong> District <strong>of</strong> Columbia) <strong>is</strong> <strong>the</strong> <strong>capital</strong>",
+            "<strong>of</strong> <strong>the</strong> United States.",
+            "It <strong>is</strong> a federal district."
+          ]
+        }
+      },
+      {
+        "_index": "my-test-data",
+        "_id": "1",
+        "_score": 0.32418242,
+        "fields": {
+          "passage_text": [
+            "Carson City is the capital city of the American state of Nevada."
+          ]
+        },
+        "highlight": {
+          "passage_text": [
+            "Carson <strong>City</strong> <strong>is</strong> <strong>the</strong> <strong>capital</strong> <strong>city</strong> <strong>of</strong> <strong>the</strong> American state <strong>of</strong> Nevada."
+          ]
+        }
+      },
+      {
+        "_index": "my-test-data",
+        "_id": "2",
+        "_score": 0.07456104,
+        "fields": {
+          "passage_text": [
+            "The Commonwealth of the Northern Mariana Islands is a group of islands in the Pacific Ocean. Its capital is Saipan."
+          ]
+        },
+        "highlight": {
+          "passage_text": [
+            "<strong>The</strong> Commonwealth <strong>of</strong> <strong>the</strong> Northern Mariana Islands <strong>is</strong> a group <strong>of</strong> islands in <strong>the</strong> Pacific Ocean.",
+            "Its <strong>capital</strong> <strong>is</strong> Saipan."
+          ]
+        }
+      },
+      {
+        "_index": "my-test-data",
+        "_id": "4",
+        "_score": 0.06124987,
+        "fields": {
+          "passage_text": [
+            "Capital punishment (the death penalty) has existed in the United States since beforethe United States was a country. As of 2017, capital punishment is legal in 30 of the 50 states."
+          ]
+        },
+        "highlight": {
+          "passage_text": [
+            "<strong>Capital</strong> punishment (<strong>the</strong> death penalty) has existed in <strong>the</strong> United States since beforethe United States",
+            "As <strong>of</strong> 2017, <strong>capital</strong> punishment <strong>is</strong> legal in 30 <strong>of</strong> <strong>the</strong> 50 states."
+          ]
+        }
+      }
+    ]
+  },
+  "profile": {
+    "shards": []
+  }
+}
+```
+
+Note: You can avoid writing the query twice by using the `query_text_path` instead of `query_text` as follows:
+```json
+POST my-test-data/_search?search_pipeline=rerank_pipeline_sagemaker
```

Comment:
Make sure to change it where you write SageMaker too

Reply:
Thanks, fixed at https://github.com/opensearch-project/ml-commons/pull/3278/commits/a219218a1cbf5930bcc871be7d7016fc7058bb03